### ISSUE TITLE
Git / Github panels missing

### ISSUE THREAD

**Issue by alexsincai** [atom/github#1004]
So after an unrelated system crash, I've noticed the Git / Github panels are missing. Hovering over the branch icon of the project shows the correct branch, the diff icon shows the appropriate number of lines added/deleted, but clicking on them does nothing, as opposed to opening the proper panes. The `CTRL + 8` / `CTRL + 9` shortcuts don't seem to be working.

Does anyone know how to get my panes back?

Thank you in advance,
Alex

**Comment by rsese** [comment]
Thanks for the report!  Are there any console errors when you try to toggle the Git/GitHub tabs?

Also, can you confirm the GitHub package is enabled in `Settings > Packages > GitHub`?

Lastly, it would be super helpful if you could fill out the issue template for future issues since it asks for a lot of baseline information:

* What version of Atom are you running (`atom -v`)?
* What operating system are you running?
* Do you have the same issue in safe mode (`atom --safe`)?

**Comment by alexsincai** [comment]
Hi, thanks for the reply.

I'm using 1.18.0 x64, on Windows 8.1 (64 bit). This issue does happen in safe mode, and yes I do have the packages installed.

Thank you

**Comment by gugadev** [comment]
Same here, Windows 10 creators update. Panels are missing, if I go to *packages -> GitHub -> Toggle GitHub Tab* nothing happen.

<details><summary><strong>StackTrace</strong></summary><p>
 
  ```
  <embedded>:69654 Error serializing package 'github' TypeError: Cannot read property 'serialize' of undefined
    at GithubPackage.serialize (C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\node_modules\github\lib\github-package.js:268:43)
    at Package.module.exports.Package.serialize (C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\src\package.js:785:103)
    at PackageManager.module.exports.PackageManager.serializePackage (C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\src\package-manager.js:712:69)
    at PackageManager.module.exports.PackageManager.serialize (C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\src\package-manager.js:705:20)
    at AtomEnvironment.module.exports.AtomEnvironment.serialize (C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\src\atom-environment.js:880:44)
    at C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\src\atom-environment.js:1194:33
    at AtomEnvironment.module.exports.AtomEnvironment.saveState (C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\src\atom-environment.js:1190:20)
    at C:\Users\gugad\AppData\Local\atom\app-1.18.0\resources\app\src\atom-environment.js:370:34
```

</p></details>
<br />

**Packages installed**

- atom-beautify
- atom-ternjs
- busy-signal
- emmet
- intentions
- language-babel
- linte
- linter-eslint
- linter-default-ui
- logo-file-icons

**Comment by rsese** [comment]
@gugadev - thanks for the stacktrace, it looks like you're running into:

https://github.com/atom/github/issues/979

There's a suggested workaround in https://github.com/atom/github/issues/979#issuecomment-311482778 if you want to give that a try.

@alexsincai - do you see the same error in console?

**Comment by andersaloof** [comment]
I've had the same issue on Windows with the latest Atom (1.19) and can confirm that changing thje environment variable NODE_ENV to 'production' (as specified in #979 and #833) fixes the issue.

**Comment by rsese** [comment]
Glad the workaround worked for you @andersaloof :+1:

Thanks again for your issue @alexsincai - we haven't received a response to our question in this comment https://github.com/atom/github/issues/1004#issuecomment-314285336 and with only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.

**Comment by andersaloof** [comment]
I did see the same error in the console, an error serializing the github package, so I guess this issue will be resolved by itself whenever #979 is resolved...
